subsys/bluetooth/mesh/adv_ext.c

```diff
@@ -235,7 +237,7 @@ static void adv_sent(struct bt_le_ext_adv *instance,
 	atomic_clear_bit(adv.flags, ADV_FLAG_ACTIVE);
 
 	if (!atomic_test_and_clear_bit(adv.flags, ADV_FLAG_PROXY)) {
-		bt_mesh_adv_send_end(0, adv.cb, adv.cb_data);
```

Comment:
I think this line needs to be reinstated like:

bt_mesh_adv_send_end(0, adv);

As it seems to not inform the calling function it has release provisioning process.

Reply:
Also above.